applications/GeoMechanicsApplication/custom_constitutive/thermal_dispersion_2D_law.hpp

```diff
@@ -0,0 +1,234 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Mohamed Nabi
+//
+
+#if !defined (KRATOS_THERMAL_DISPERSION_2D_LAW_GEO_H_INCLUDED)
+#define  KRATOS_THERMAL_DISPERSION_2D_LAW_GEO_H_INCLUDED
```

Comment:
#pragma once

Reply:
do you mean I need to remove lines 13 and 14, and replace them by #pragma once ?

Reply:
Done